src/haz3lweb/view/SchoolMode.re

```diff
@@ -340,43 +340,49 @@ let view =
       ]
       : [];
 
-  div(
-    ~attr=Attr.classes(["editor", "column"]),
-    [title_view, prompt_view]
-    @ render_cells(
-        settings,
-        [
-          prelude_view,
-          correct_impl_view,
-          correct_impl_ctx_view,
-          your_tests_view,
-        ]
-        @ wrong_impl_views
-        @ [
-          mutation_testing_view,
-          your_impl_view,
-          impl_validation_view,
-          hidden_tests_view,
-          impl_grading_view,
-        ],
-      )
-    // TODO lang doc visibility tied to ci visibility (is this desired?)
-    @ [div(~attr=Attr.class_("bottom-bar"), ci_view)]
-    @ (
-      langDocMessages.show && settings.statics
-        ? [
-          LangDoc.view(
-            ~inject,
-            ~font_metrics,
-            ~settings,
-            ~doc=langDocMessages,
-            Indicated.index(focal_zipper),
-            focal_info_map,
+  [
+    div(
+      ~attr=Attr.classes(["main"]),
+      [
+        div(
+          ~attr=Attr.classes(["editor", "column"]),
+          [title_view, prompt_view]
+          @ render_cells(
+              settings,
+              [
+                prelude_view,
+                correct_impl_view,
+                correct_impl_ctx_view,
+                your_tests_view,
+              ]
+              @ wrong_impl_views
+              @ [
+                mutation_testing_view,
+                your_impl_view,
+                impl_validation_view,
+                hidden_tests_view,
+                impl_grading_view,
+              ],
+            )
+          @ (
+            langDocMessages.show && settings.statics
+              ? [
+                LangDoc.view(
+                  ~inject,
+                  ~font_metrics,
+                  ~settings,
+                  ~doc=langDocMessages,
+                  Indicated.index(focal_zipper),
+                  focal_info_map,
+                ),
+              ]
+              : []
           ),
-        ]
-        : []
+        ),
+      ],
     ),
-  );
+    div(~attr=Attr.class_("bottom-bar"), ci_view),
+  ]; // TODO lang doc visibility tied to ci visibility (is this desired?)
```

Comment:
I tried to keep the changes to a minimum to avoid merge conflicts. These now return a list of divs that can be added to the flexbox div.

Reply:
yes tying langdoc visibility and ci visibility is fine since the toggle for langdocs is in the ci

Reply:
I had just moved the comment since the code got slightly rearranged. I'll probably leave it unless you think it's better removed.